Contribution Guide.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Flutter Devs Kenya 🚀
+
+Welcome to the Flutter Devs Kenya community! We're excited that you're interested in contributing to our project. Your contributions play a vital role in making our community and resources better.
+
+## How to Contribute
+
+### Code Contributions
+
+If you're a developer, you can contribute by improving our website, app, or related projects. Here's how you can get started:
+
+1. Fork the repository to your own GitHub account.
+2. Create a new branch with a descriptive name.
+3. Make your changes and improvements.
+4. Ensure your code follows our Coding Guidelines.
+5. Submit a pull request to the main branch of the original repository.
+6. Your pull request will be reviewed by the project maintainers.
+
+### Content Contributions
+
+We welcome contributions to our documentation, tutorials, and any other content related to Flutter development. Here's how you can contribute:
+
+1. Fork the repository to your own GitHub account.
+2. Create a new branch with a descriptive name.
+3. Make your content changes and improvements.
+4. Ensure your content follows our Content Guidelines.
+5. Submit a pull request to the main branch of the original repository.
+6. Your pull request will be reviewed by the project maintainers.
+
+### Reporting Issues
+
+If you come across any issues, bugs, or have suggestions for improvements, please create a new issue on our GitHub repository. We appreciate your feedback!
+
+## Getting Started
+
+Before contributing, make sure you have the following:
+
+- A GitHub account.
+- Knowledge of Flutter development (if contributing code).
+- Familiarity with Markdown (if contributing documentation).
+
+## Coding Guidelines
+
+When contributing code, please adhere to the following guidelines:
+
+- Include appropriate comments and documentation.
+- Write clear and concise commit messages.
+- Test your code thoroughly.
+
```

Comment:
We can also add to follow the [effective dart style guide](https://dart.dev/effective-dart)

Reply:
Are you still interested in this?